docs-md/tutorials/embedded-connect.md

```diff
@@ -0,0 +1,409 @@
+configuration file:
+
+```bash
+cat << EOF > ./connect.properties
+bootstrap.servers=broker:9092
+plugin.path=/usr/share/kafka/plugins
+group.id=ksql-connect-cluster
+key.converter=org.apache.kafka.connect.json.JsonConverter
+value.converter=org.apache.kafka.connect.json.JsonConverter
+value.converter.schemas.enable=false
+internal.key.converter.schemas.enable=false
+config.storage.topic=ksql-connect-configs
+offset.storage.topic=ksql-connect-offsets
+status.storage.topic=ksql-connect-statuses
+config.storage.replication.factor=1
+offset.storage.replication.factor=1
+status.storage.replication.factor=1
+EOF
+```
+
+4. Start ksqlDB and PostgreSQL
+------------------------------
+
+In the directory containing the `docker-compose.yml` file you created in the
+first step, run the following command to start all services in the correct
+order.
+
+```bash
+docker-compose up
+```
+
+5. Connect to PostgreSQL
+------------------------
+
+Run the following command to establish an interactive session with PostgreSQL.
+
+```bash
+docker exec -it postgres psql -U postgres
+```
+
+6. Populate PostgreSQL with vehicle/driver data
+-----------------------------------------------
+
+In the PostgreSQL session, run the following SQL statements to set up the
+driver data. You will join this PostgreSQL data with event streams in ksqlDB.
+
+```sql
+CREATE TABLE drivers (
+  driver_id integer PRIMARY KEY,
+  make text,
+  model text,
+  year integer,
+  license_plate text,
+  rating float
+);
+
+INSERT INTO drivers (driver_id, make, model, year, license_plate, rating) VALUES
+  (0, 'Toyota', 'Prius',   2019, 'KAFKA-1', 5.00),
+  (1, 'Kia',    'Sorento', 2017, 'STREAMS', 4.89),
+  (2, 'Tesla',  'Model S', 2019, 'CNFLNT',  4.92),
+  (3, 'Toyota', 'Camry',   2018, 'ILVKSQL', 4.85);
+```
+
+7. Start ksqlDB's interactive CLI
+---------------------------------
+
+ksqlDB runs as a server which clients connect to in order to issue queries.
+
+Run the following command to connect to the ksqlDB server and start an
+interactive command-line interface (CLI) session.
+
+```bash
+docker exec -it ksqldb-cli ksql http://ksqldb-server:8088
+```
+
+8. Create source connector
+--------------------------
+
+Make your PostgreSQL data accessible to ksqlDB by creating a *source*
+connector. In the ksqlDB CLI, run the following command.
+
+```sql
+CREATE SOURCE CONNECTOR jdbc_source WITH (
+  'connector.class'          = 'io.confluent.connect.jdbc.JdbcSourceConnector',
+  'connection.url'           = 'jdbc:postgresql://postgres:5432/postgres',
+  'connection.user'          = 'postgres',
+  'topic.prefix'             = 'jdbc_',
+  'table.whitelist'          = 'drivers',
+  'mode'                     = 'incrementing',
+  'numeric.mapping'          = 'best_fit',
+  'incrementing.column.name' = 'driver_id',
+  'key'                      = 'driver_id',
+  'value.converter.schemas.enable' = false);
+
+```
+
+When the source connector is created, it imports any PostgreSQL tables matching
```

Comment:
Would it be worth mentioning the special `key` config property on the connector? This is unique to ksqlDB.